examples/example1.jl

```diff
@@ -16,9 +19,9 @@ add!(m, Commodity(:PK))
 
 add!(m, Consumer(:RA, benchmark=150.))
 
-add!(m, Production(:X, 1, :PX, 100, [Input(:PL, 50), Input(:PK, 50)]))
-add!(m, Production(:Y, 1, :PY, 50, [Input(:PL, 20), Input(:PK, 30)]))
-add!(m, Production(:U, 1, :PU, 150, [Input(:PX, 100), Input(:PY, 50)]))
+add!(m, Production(:X, 1., :PX, 100., [Input(:PL, :(50 * $inputcoeff)), Input(:PK, 50)]))
+add!(m, Production(:Y, :(1 * $elascoeff), :PY, 50., [Input(:PL, 20), Input(:PK, 30)]))
+add!(m, Production(:U, 1., :PU, :(150 * $outputmult), [Input(:PX, 100), Input(:PY, 50)]))
```

Comment:
I think in general I would put the code that tests this into the tests, not into the examples. We want the examples to reflect the original example, I think?

Also, just in general, adding tests is more useful because then that code runs often and we learn earlier if we break something.

Reply:
Resolved (I hope) in the next commit. I reverted the example 1, and added the coefficients to test expressions in the test instead.